src/banking/sheba.rs

```diff
@@ -0,0 +1,356 @@
+use crate::impl_trait_for_string_types;
+use lazy_static::lazy_static;
+use regex::Regex;
+
+lazy_static! {
+    static ref SHEBA_CODE_PATTERN: Regex = Regex::new(r#"^IR[0-9]{24}$"#).unwrap();
+}
+
+#[derive(Debug)]
+pub struct ShebaInfo {
+    nickname: &'static str,
+    name: &'static str,
+    persian_name: &'static str,
+    code: &'static str,
+    account_available: bool,
```

Comment:
What is this `account_available`?

Reply:
If you understand, please tell me ;)

Reply:
lol, so then remove it :D 
everywhere it's `false` and we don't edit it!